docs/data-sources/cloud_backup_schedule.md

```diff
@@ -33,18 +33,40 @@ resource "mongodbatlas_cloud_backup_schedule" "test" {
   reference_hour_of_day    = 3
   reference_minute_of_hour = 45
   restore_window_days      = 4
+
+  policy_item_daily {
+    frequency_interval = 1
+    retention_unit     = "days"
+    retention_value    = 14
+  }
+
+  copy_settings {
+    cloud_provider = "AWS"
+    frequencies = ["HOURLY",
+		   "DAILY",
+		   "WEEKLY",
+		   "MONTHLY",
+		   "YEARLY",
+		   "ON_DEMAND"]
+    region_name = "US_EAST_1"
+    zone_id = mongodbatlas_advanced_cluster.my_cluster.replication_specs.*.zone_id[0]
+    should_copy_oplogs = false
+  }
 }
 
 data "mongodbatlas_cloud_backup_schedule" "test" {
   project_id   = mongodbatlas_cloud_backup_schedule.test.project_id
   cluster_name = mongodbatlas_cloud_backup_schedule.test.cluster_name
+  use_zone_id_for_copy_settings = true
 }
 ```
 
 ## Argument Reference
 
 * `project_id` - (Required) The unique identifier of the project for the Atlas cluster.
 * `cluster_name` - (Required) The name of the Atlas cluster that contains the snapshots backup policy you want to retrieve.
+* `use_zone_id_for_copy_settings` - Set this field to `true` to allow the data source to use the latest schema that populates `copy_settings.#.zone_id` instead of the deprecated `copy_settings.#.replication_spec_id`. These also enables you to reference cluster zones using independent shard scaling, which no longer supports `replication_spec.*.id`. To learn more, see the [1.18.0 upgrade guide](../guides/1.18.0-upgrade-guide.md#transition-cloud-backup-schedules-for-clusters-to-use-zones).
```

Comment:
```suggestion
* `use_zone_id_for_copy_settings` - Set this field to `true` to allow the data source to use the latest schema that populates `copy_settings.#.zone_id` instead of the deprecated `copy_settings.#.replication_spec_id`. These fields also enable you to reference cluster zones using independent shard scaling, which no longer supports `replication_spec.*.id`. To learn more, see the [1.18.0 upgrade guide](../guides/1.18.0-upgrade-guide.md#transition-cloud-backup-schedules-for-clusters-to-use-zones).
```